Update USAGE.md w/more clear relaunch instructions

andrew-dawson-seqera opened the pull request.

andrew-dawson-seqera committed:
Update USAGE.md w/more clear relaunch instructions

robnewman commented on USAGE.md:
```diff
@@ -375,6 +375,8 @@ $ tw runs list --filter="*man/rnaseq-*"
 
 Run `tw runs relaunch -h` to view all the required and optional fields for relaunching a run in a workspace.
 
+**TL;DR:** This is how you relaunch a run: `tw runs relaunch -i <RUN_ID> --workspace <WORKSPACE_NAME>`.
```
You can't use the expanded --workspace without an equals sign. Updating to reflect other examples.

```suggestion
In the example below, we relaunch a run with ID `3S3KzAasBa4Bt3` in workspace `my-workspace`:
```console
tw runs relaunch -i 3S3KzAasBa4Bt3 -w my-workspace
```

robnewman reviewed (changes requested):
Fixing syntax and align with other examples

andrew-dawson-seqera committed:
Update USAGE.md

Co-authored-by: Rob Newman <61608+robnewman@users.noreply.github.com>

robnewman commented on USAGE.md:
````diff
@@ -375,6 +375,10 @@ $ tw runs list --filter="*man/rnaseq-*"
 Run `tw runs relaunch -h` to view all the required and optional fields for relaunching a run in a workspace.
 
+In the example below, we relaunch a run with ID `3S3KzAasBa4Bt3` in workspace `my-workspace`:
+```console
````
```suggestion

```console
```

robnewman commented on USAGE.md:
````diff
@@ -375,6 +375,10 @@ $ tw runs list --filter="*man/rnaseq-*"
+In the example below, we relaunch a run with ID `3S3KzAasBa4Bt3` in workspace `my-workspace`:
+```console
+tw runs relaunch -i 3S3KzAasBa4Bt3 -w my-workspace
+
````
I believe you need a trailing ` ``` ` and newline to cleanly close off the `console` code block.

robnewman reviewed (approved):
Approved subject to very minor changes being incorporated.

jordeu committed:
Close fenced code block and add shell prompt

Without the closing ``` fence, the rest of USAGE.md from "### Cancel a
run" onward was rendered as code. Adds the missing fence and prefixes
the example with $ to match the convention used elsewhere in the file.

jordeu committed:
Fix markdownlint MD014 and MD031

- MD031: add blank line before the fenced code block
- MD014: drop the $ prefix since the example doesn't show output
